README.md

```diff
@@ -0,0 +1,93 @@
+# Spotify Web SDK
+
+A JavaScript SDK for the [Spotify Web API](https://developer.spotify.com/documentation/web-api/).
+
+## Features
+
+### "Why should I use this?"
+
+We want to handle the hard work for you.
+Even to make simple requests, such as to get information on a track, you'd need to make a bunch of setups.
+Here is how you could do the same thing with our aid:
+
+```javascript
+import { init, getTrack } from 'spotify-web-sdk';
```

Comment:
I tested import this way on `start.ts` and didn't work, because it's an `export default`, so you have to export this way
```suggestion
import spotify from 'spotify-web-sdk';
```